### PR TITLE
[LUA] dhalmel meat enum mismatch

### PR DESCRIPTION
**Pull request by MowFord**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

The enum for dhalmel meat doesn't match in all but I've place in the code. Perhaps the issue is the items.lua?

## Steps to test these changes

This PR is opened specifically because the kolshushi regional vendor in bastok was failing to load its shop, but I found the other instances as well. Would you like me to whip up a script for CI to catch this globally?

### CODE REVIEW

File: scripts/items/behemoth_suit_+1.lua [hunk 1]
```diff
@@ -24,7 +24,7 @@ itemObject.onItemUse = function(target)
         {  4, xi.item.SLICE_OF_CERBERUS_MEAT    },
         { 17, xi.item.SLICE_OF_COCKATRICE_MEAT  },
         { 13, xi.item.SLICE_OF_COEURL_MEAT      },
-        { 20, xi.item.DHALMEL_MEAT              },
+        { 20, xi.item.SLICE_OF_DHALMEL_MEAT              },
```
Comment (TeoTwawki) [inline scripts/items/behemoth_suit_+1.lua]: gotta realign closing brace

Comment (MowFord) [reply scripts/items/behemoth_suit_+1.lua]: Damn sorry

**Review by TeoTwawki — APPROVED** [review]

**Comment by WinterSolstice8** [comment]
nitpick, but `enun` in the commit title

**Review by WinterSolstice8 — APPROVED** [review]

File: scripts/zones/Bastok_Markets/npcs/Yafafa.lua [hunk 2]
```diff
@@ -18,7 +18,7 @@ entity.onTrigger = function(player, npc)
         {
             xi.item.BULB_OF_MHAURA_GARLIC,      83,
             xi.item.YAGUDO_CHERRY,              45,
-            xi.item.DHALMEL_MEAT,              249,
+            xi.item.SLICE_OF_DHALMEL_MEAT,              249,
```
Comment (Xaver-DaRed) [inline scripts/zones/Bastok_Markets/npcs/Yafafa.lua:21]: please, realign this.

Rest seems fine

Comment (Xaver-DaRed) [inline scripts/zones/Bastok_Markets/npcs/Yafafa.lua]: Please re-align this.

The rest of the changes -> :cheffkiss:

**Review by Xaver-DaRed — CHANGES_REQUESTED** [review]

File: scripts/zones/Bastok_Markets/npcs/Yafafa.lua (hunk 2 above)
Comment (MowFord) [reply scripts/zones/Bastok_Markets/npcs/Yafafa.lua]: Sorry about that, All done

**Review by Xaver-DaRed — APPROVED** [review]

**Review by EpicTaru — APPROVED** [review]

**Pull request merged**